File: Api/Companies/iCompanyPortal.Api.Companies.Client/Company.cs
using System;
using System.Collections.Generic;

namespace iCompanyPortal.Api.Companies.Client
{
    public class Company
    {
        public int CompanyId { get; set; }

        public string Name { get; set; }

        public string UniqueName { get; set; }

        public CompanyStatus Status { get; set; }

        public DateTime CreatedAt { get; set; }

        public DateTime? DeleteAt { get; set; }

        public byte[] Logo { get; set; }

        public string LogoContentType { get; set; }

        public ICollection<CompanyInvitation> CompanyInvitations { get; set; } = new HashSet<CompanyInvitation>();

        public ICollection<CompanyUser> CompanyUsers { get; set; } = new HashSet<CompanyUser>();

        public Subscription Subscription { get; set; }
    }
}

File: App/icompanyportal/projects/companies-api/src/lib/models/company.ts
import { CompanyStatus } from './company-status';
import { Subscription } from './subscription';

export interface Company {
  companyId?: number;
  uniqueName?: string;
  name?: string;
  status?: CompanyStatus;
  logo?: string;
  logoContentType?: string;

  subscription?: Subscription;
}
